service/worker/migration/activities.go

```diff
@@ -697,23 +700,13 @@ func (a *activities) VerifyReplicationTasks(ctx context.Context, request *verify
 		}
 
 		diff := time.Now().Sub(details.CheckPoint)
-		if diff > defaultNoProgressRetryableTimeout {
-			if diff > defaultNoProgressNotRetryableTimeout {
-				// Potentially encountered a missing execution, return non-retryable error
-				return temporal.NewNonRetryableApplicationError(
-					fmt.Sprintf("verifyReplicationTasks was not able to make progress for more than %v minutes (not retryable). Not found WorkflowExecution: %v, Checkpoint: %v",
-						diff.Minutes(),
-						details.LastNotFoundWorkflowExecution, details.CheckPoint),
-					"", nil)
-			}
-
-			// return error to trigger activity retry
-			return verifyReplicationTasksTimeoutErr{
-				timeout: diff,
-				details: details,
-			}
+		if diff > defaultNoProgressNotRetryableTimeout {
```

Comment:
`defaultNoProgressRetryableTimeout` is not needed any more. it only leads to unnecessary activity retry.